ios/KaMPStarteriOS/ViewController.swift

```diff
@@ -9,12 +9,12 @@
 import UIKit
 import shared
 
-class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
Did we lose `UITableViewDelegate` entirely?

Reply:
Yeah, UITableViewDelegate handles things like selecting/editing rows, cell height, header/footer stuff, etc... All things we're not doing because the sample app is so barebones. The only UITableView functions we use are in UITableViewDataSource.